File: SlotRead.js
function SlotRead() {
  this._slot = null;
}
SlotRead.prototype.getSlot = function() {
  return this._slot;
}
SlotRead.prototype.setSlot = function(v) {
  this._slot = v;
  return this;
}
SlotRead.prototype.bench = function() {
  this.setSlot('something');
  for (var i = 0; i < 100000; i++) {
    this.getSlot();
    this.getSlot();
    this.getSlot();
    this.getSlot();
    this.getSlot();
    this.getSlot();
    this.getSlot();
    this.getSlot();
    this.getSlot();
    this.getSlot();
  }
}
global.SlotRead = SlotRead;

File: SlotRead.dart
class SlotRead {
  var _slot;
  getSlot() {
    return this._slot;
  }
  setSlot(v) {
    this._slot = v;
    return this;
  }
  bench() {
    this.setSlot('something');
    for(var i = 0; i < 100000; i++) {
      this.getSlot();
      this.getSlot();
      this.getSlot();
      this.getSlot();
      this.getSlot();
      this.getSlot();
      this.getSlot();
      this.getSlot();
      this.getSlot();
      this.getSlot();
    }
  }
}

